stable_baselines3/common/env_util.py

```diff
@@ -138,22 +138,16 @@ def make_atari_env(
     :param monitor_kwargs: Keyword arguments to pass to the ``Monitor`` class constructor.
     :return: The wrapped environment
     """
-    if wrapper_kwargs is None:
-        wrapper_kwargs = {}
-
-    def atari_wrapper(env: gym.Env) -> gym.Env:
```

Comment:
I need to check if there was a reason I did it that way

Reply:
that's fine, it was because `wrapper_kwargs` was introduced only later: https://github.com/DLR-RM/stable-baselines3/pull/448